src/server/status/metrics_collector/metrics.test.js

```diff
@@ -24,30 +38,38 @@ describe('Metrics', function () {
   });
 
   afterEach(() => {
-    mockFs.restore();
+    setMockFiles();
   });
 
 
   describe('capture', () => {
     it('merges all metrics', async () => {
-      mockFs();
-      sinon.stub(metrics, 'captureEvent').returns({ 'a': [{ 'b': 2 }, { 'd': 4 }] });
+      setMockFiles();
+      sinon.stub(metrics, 'captureEvent').returns({ 'a': [{ 'b': 2 }, { 'd': 4 }], process: { uptime_ms: 1980 } });
       sinon.stub(metrics, 'captureCGroupsIfAvailable').returns({ 'a': [{ 'c': 3 }, { 'e': 5 }] });
       sinon.stub(Date.prototype, 'toISOString').returns('2017-04-14T18:35:41.534Z');
-      sinon.stub(process, 'uptime').returns(5000);
```

Comment:
this was existing code, but it looks like stubbing `uptime` this way actually affects all the tests. That's why this had to be moved down to `parses the hapi event`